src/WHAT_TO_DO_NEXT.md

```diff
@@ -0,0 +1,135 @@
+```
+
+`build.ps1` can be used to build and test the module.
+
+Build the module: `build.ps1 -Build`
+Test the module: `build.ps1 -Test`
+
+### Deep-dive into individual folders
+
+#### Source code
+
+The scaffolding that is generated has stubs for both script cmdlet as well as binary cmdlet.
+The module writer should delete the `code` folder if the module is a script module.
+
+```PowerShell
+
+    Directory: D:\temp\MyTestMod\src
+
+Mode                 LastWriteTime         Length Name
+----                 -------------         ------ ----
+d----           7/29/2019  4:08 PM                code
+-a---           7/29/2019  4:08 PM           4174 MyTestMod.psd1
+-a---           7/29/2019  4:08 PM              0 MyTestMod.psm1
+
+    Directory: D:\temp\MyTestMod\src\code
+
+Mode                 LastWriteTime         Length Name
+----                 -------------         ------ ----
+-a---           7/29/2019  4:08 PM            353 Class1.cs
+-a---           7/29/2019  4:08 PM            253 MyTestMod.csproj
+
+```
+
+#### Test code
+
+The sample test is generated which can be executed using `build.ps1`.
+
+```PowerShell
+
+    Directory: D:\temp\MyTestMod\test
+
+Mode                 LastWriteTime         Length Name
+----                 -------------         ------ ----
+-a---           7/29/2019  4:08 PM            281 MyTestMod.Tests.ps1
+
+```
+
+#### Help content
+
+The MD file for `about_Help` topic is created.
+Cmdlet help can be generated when the module is built.
+
+```PowerShell
+
+    Directory: D:\temp\MyTestMod\help
+
+Mode                 LastWriteTime         Length Name
+----                 -------------         ------ ----
+d----           7/29/2019  4:08 PM                en-US
+
+    Directory: D:\temp\MyTestMod\help\en-US
+
+Mode                 LastWriteTime         Length Name
+----                 -------------         ------ ----
+-a---           7/29/2019  4:08 PM           1460 about_MyTestMod.md
+
+```
+
+#### CI templates
+
+The YAML files are stubs that are need to configure CI/CD pipelines on Azure DevOps.
+
+```PowerShell
+    Directory: D:\temp\MyTestMod\.ci
+
+Mode                 LastWriteTime         Length Name
+----                 -------------         ------ ----
+-a---           7/29/2019  3:45 PM           1855 ci.yml
+-a---           7/29/2019  3:45 PM           2028 release.yml
+-a---           7/29/2019  3:45 PM           2028 test.yml
+
+```
+
+## Build customization
+
+The generated `build.ps1` file has a function called `DoBuild`.
+Customize this function to suit your build needs.
+When the function is executed, the expectation is that the `BuildOutputPath` as a sub-folder for the module with all required assemblies and script assests.
+
+## Enable CI in Azure DevOps
+
+For maintaining quality of the module, it is highly recommended that a CI system should be used.
+We recommend using Azure DevOps, and hence assist onboarding by creating YAML files for the same.
```

Comment:
We provide YAML files which are ready for use by AzureDevOps to accelerate the automated build, test, and release phases.